install/kubernetes/cilium/values.yaml

```diff
@@ -2029,6 +2042,15 @@ operator:
       relabelings: ~
       # -- Metrics relabeling configs for the ServiceMonitor cilium-operator
       metricRelabelings: ~
+  dashboards:
+    # -- Grafana dashboards for cillium-operator
+    # grafana can import dashboards based on the label and value
+    # ref: https://github.com/grafana/helm-charts/tree/main/charts/grafana#sidecar-for-dashboards
+    enabled: false
```

Comment:
For the operator and hubble, the doc comment is above the `enabled` field. For cilium-agent, it's above the `dashboard` field. I think it's best if they are all consistently above the `dashboard` field.

Reply:
This look better?

Reply:
Yes, thanks!